operator/config/overlays/openshift/controller_manager_config.yaml

```diff
@@ -38,7 +38,7 @@ featureGates:
   #
   lokiStackGateway: true
   grafanaLabsUsageReport: false
-  runtimeSeccompProfile: false
+  restrictedPodSecurityStandard: true
```

Comment:
Question, are these four different `controller_manager_config.yaml` files supposed to have different  `restrictedPodSecurityStandard` settings?  Two are set to `true` and two are set to `false`.